3D-microservices-observability/node-js-app/beachshirt/controllers/stylingController.js

```diff
@@ -0,0 +1,47 @@
+/**
+* Driver for styling service which manages different styles of shirts and takes orders for a shirts
+* of a given style.
+* @author Yogesh Prasad Kurmi (ykurmi@vmware.com)
+*/
+const request = require('request');
+const uuidv1 = require('uuid/v1');
+
+module.exports = function(app, config, log){
+    app.get('/style', function(req, res){
+        res.json(
+            [
+                {
+                    "name": "Wavefront",
+                    "url": "WavefrontURL"
+                },
+                {
+                    "name": "BeachOps",
+                    "url": "BeachOpsURL"
+                }
+            ]
+        );
+    });
+
+    app.get('/style/:id/make', function(req, res){
+        if (Math.floor(Math.random() * 10) == 0) {
+            msg = "Failed to order shirts!";
+            log.error(msg);
+            return res.status(503).json({ error: msg });
+        }
+        var shirts = [];
+        var i;
+        for(i = 0; i < parseInt(req.query.quantity); i++){
```

Comment:
Consider using let and const to define variables instead of var.  Not sure what version of NodeJs we intend to support with this sample code, but maybe change functions to fat arrow as well.  Saves a lot of headaches down the road.